labrad/backend.py

```diff
@@ -45,6 +47,13 @@ def disconnect(self):
             self._disconnect()
             self.connected = False
 
+    def spawn(self):
+        """Start a new independent backend connection to the same manager."""
+        inst = self.__class__(name=self.name)
```

Comment:
@maffoo ah I see. I think you could make the code a little clearer for the reader, or add a comment as to why you do this.